markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -183,6 +183,24 @@ library PerpsAccount {
         }
     }
 
+    function getMarginLiquidationCost(
+        Data storage self
+    ) internal returns (uint256 marginLiquidateCost, uint256 seizedMarginValue) {
+        // SetUtil.UintSet storage liquidatableAccounts = GlobalPerpsMarket
+        //     .load()
+        //     .liquidatableAccounts;
+
+        // if (!liquidatableAccounts.contains(self.id)) {
+        // notice: using getFlagKeeperCosts here since the logic is the same, but with no positions.
+        marginLiquidateCost = KeeperCosts.load().getFlagKeeperCosts(self.id);
+        // TODO do we need to add it to flagged accounts?
```

Comment:
no, flaggedAccounts is only used for accounts with positions for rate limited liquidations.  for margin, we can fully execute the liquidation in one go so no need to keep track of anything else.